Dockerfile.nginx-alpine

```diff
@@ -5,9 +5,9 @@
 # Dockerfile.django-alpine to use the caching mechanism of Docker.
 
 # Ref: https://devguide.python.org/#branchstatus
-FROM node:23.7.0-alpine3.20@sha256:3ac002b133fd3737c44b66f0d023f5d75c7a1ddec71954fa06486aed6aead888 AS node
+FROM node:23.7.0-alpine3.21@sha256:70eca392e3d52cb7d133b52d52e8600d8c410a5eaee6105c11324b28868f9ac9 AS node
 
-FROM python:3.11.9-alpine3.20@sha256:f9ce6fe33d9a5499e35c976df16d24ae80f6ef0a28be5433140236c2ca482686 AS base
+FROM python:3.11.11-alpine3.20@sha256:6e18772230b36e78251ed179a2a2a2b3cc94726f02e1fddccdcfbe05b17bdc96 AS base
```

Comment:
Originally I wanted to go up to `alpine3.21` but it is failing with 
```
 > [nginx collectstatic 2/8] RUN npm install -g yarn --force:
0.280 Error relocating /usr/bin/node: sqlite3session_attach: symbol not found
0.280 Error relocating /usr/bin/node: sqlite3changeset_apply: symbol not found
0.280 Error relocating /usr/bin/node: sqlite3session_create: symbol not found
0.280 Error relocating /usr/bin/node: sqlite3session_changeset: symbol not found
0.281 Error relocating /usr/bin/node: sqlite3session_patchset: symbol not found
0.281 Error relocating /usr/bin/node: sqlite3session_delete: symbol not found
```
It probably needs some additional customization. I will solve it in a separated PR.

Reply:
During my arm64 testing I also noticed some issues on pythong 3.12.x / alpine 3.21. And on arm64 I also see issues with sqlite.